src/gsi/setupq.f90

```diff
@@ -394,7 +400,11 @@ subroutine setupq(obsLL,odiagLL,lunin,mype,bwork,awork,nele,nobs,is,conv_diagsav
      ioff0=21
      nreal=ioff0
      if (lobsdiagsave) nreal=nreal+4*miter+1
-     if (twodvar_regional) then; nreal=nreal+2; allocate(cprvstg(nobs),csprvstg(nobs)); endif
+     if (twodvar_regional .or. l_obsprvdiag) then
+       nreal=nreal+2                            ! account for idomsfc,izz
+       allocate(cprvstg(nobs),csprvstg(nobs))   ! obs provider info
+       if(l_pbl_pseudo_surfobsq) allocate(cprvstgp(nobs*3),csprvstgp(nobs*3)) ! obs provider info for pseudo obs
+       endif
```

Comment:
remove two spaces before line 407 so that it aligns with line 403

Reply:
@guoqing-noaa @hu5970
Thank you for the review. 
The two spaces in beginning of the line 407 of "setupq/f90" was removed. I make a new commit and push it back. 

As for comparing with old code for validation, I have not run it yet.  My colleague Matthew Morris is going to test it. I will let you know the results of comparison. But he will only test with HRRR-based run, not for FV3-LAM based rrfs run.

Thank you!